CHARTER.md

```diff
@@ -0,0 +1,237 @@
+# JSON Schema Org Charter
+<!-- This document is managed in the json-schema-org/community GitHub repository. Please do NOT modify this file in another repository as changes may be overridden. -->
+
+## Section 0: Guiding Principles
+The JSON Schema project is part of the OpenJS Foundation which operates transparently, openly, collaboratively, and ethically. We strive to be open and transparent as much as is possible, and wish to enable anyone to interact and engage with any area of our work.
+
+Having no structure in place usually leads to one that is informal and undocumented, making it difficult to meet our own expectations of how we wish to operate. As such, we define the following charter which includes aspects of the governance model to which we subscribe and by which we operate.
+
+## Section 1: Scope
+JSON Schema aims to enable the confident and reliable use of the JSON data format. It does this primarily by providing specification documents which define a declarative language that allows annotation and validation of JSON documents.
+While JSON Schema's primary target is constraint-based data validation, it continues to be used across the whole stack, in different stages of a given project, and for purposes beyond its original design. We aim to enable these additional and emergent use cases.
+
+### 1.1: In-scope
+The scope of the JSON Schema project is split into two sections: primary and secondary concerns.
+Primary concerns are areas we wish to give focus to, while secondary concerns are more likely to require the creation of a Special Interest Group.
+
+Primary Concerns
+- Publication of the JSON Schema standard
+  - Validation of JSON data
+  - Semantic annotation of JSON data
+  - Extensibility
+- Critical tooling
+  - Creation of new
+  - Ingestion of existing
+  - Long term support
+- Documentation
+- Test suite
+- Community
+  - Enabling schema authors
+  - Enabling implementers
+  - Engaging with industry
+  - Communicating value
+  - Ensuring the sustainability of the project
+
+Secondary Concerns
+- Hypermedia
+- Generating JSON Schema
+- Using JSON Schema to generate
+  - Code (including types or classes)
+  - UI (including forms)
+  - Databases
+- Relational validation
+- Vocabularies registry
+
+### 1.2: Out-of-Scope
+Section Intentionally Left Blank
```

Comment:
I might be tempted to include something to do with "decisions made in related projects with critical dependencies on JSON Schema such as Open API"; while we'd offer advice and input, and recommended practices as with any other community project, we're not responsible for those decisions.

Reply:
Ideally there would be at least one other project in that list of "examples of projects that are not our problem"

Reply:
> Does this kind of statement maybe belong under section 2.1 "Other Formal Project Relationships"?

I'd say no, as there are no formal relationships. We may make some formal relationships later though which would be neat!

Reply:
> I might be tempted to include something to do with "decisions made in related projects with critical dependencies on JSON Schem... - @mwadams 

We discussed this on last nights [OCWM call](https://github.com/json-schema-org/community/issues/374). There were concerns it sounded rather abrasive. I'm going to try something that sounds less combative and more positive.

Reply:
@mwadams @gregsdennis I've pushed a change. Let me know what you think =]

Reply:
I like it. I've already approved.